core/storage/suggestion/gae_models.py

```diff
@@ -629,6 +634,33 @@ def get_in_review_question_suggestions_by_offset(
                 next_offset: int. The input offset + the number of results
                     returned by the current query.
         """
+
+        if sort_key == 'Date':
```

Comment:
'Date' should be a named constant and declared just once in the entire codebase (probably in constants.ts). I.e. there should be a single place where it's defined and if you change its value there, the code should still work with no issues.